build.sbt

```diff
@@ -67,18 +70,21 @@ lazy val utilInterface = (project in internalPath / "util-interface").
     commonSettings,
     javaOnlySettings,
     name := "Util Interface",
-    exportJars := true
+    exportJars := true,
+    mimaSettings,
```

Comment:
```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scala-sbt#util-interface_2.12;1.0.0-M28: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

util-interface is not cross published. Do you need to do something special here?